### PR TITLE
13 fe lobby page

### LINKED ISSUE #13: [FE] Lobby page

### PR DESCRIPTION
**Pull request by Suntoh**
## What I did

 - create a lobby page
 - fix some login page
 - create navigation bar

## Note

waiting for socket emit&on

### CODE REVIEW

File: apps/client/src/app/globals.css [hunk 1]
```diff
@@ -8,6 +8,23 @@
...
+  width: 100vw; /* Full viewport width */
+  position: fixed; 
+=======
+  background-position: center; /* Centers the background */
+  background-repeat: no-repeat; /* Prevents tiling */
+  height: 100vh; /* Ensures it takes the full viewport height */
+  width: 100vw; /* Full viewport width */
+>>>>>>> 1b6c80e (feat: background images, Navbar)
+}
+
```
Comment (Imeany12) [inline apps/client/src/app/globals.css]: Isn't this a conflict message? Please resolve them duplicate properties.

File: apps/client/src/app/page.tsx [hunk 2]
```diff
@@ -13,52 +14,50 @@ export default function Home() {
...
+            <Link href='/api/auth/signin' className='text-3xl text-red-500'>
+              Sign In
+            </Link>
+          </div>
+        )}
+        <Link href='myAccount' className='text-2xl text-green-600'>
+          My Account
+        </Link>
+        {!mode ? (
+          <div className='flex h-screen flex-row items-end'>
```
Comment (Imeany12) [inline apps/client/src/app/page.tsx]: This bad boy `h-screen` make the home page's height behave in weird manner
Screenshot 2567-10-12 at 10 29 58

**Review by Imeany12 — CHANGES_REQUESTED** [review]

File: apps/client/src/app/page.tsx (hunk 2 above)
Comment (Suntoh) [reply apps/client/src/app/page.tsx]: i think in gino's work he has fixed this already so i didn't deal with it i will just delete it and wait for gino

Comment (Suntoh) [reply apps/client/src/app/page.tsx]: I change it to  an OK one.

**Comment by Imeany12** [comment]
LGTM 👍 👍

**Review by Imeany12 — APPROVED** [review]
LGTM 👍 👍

**Pull request merged**